cpp/meta_module/algorithm/meta.cpp

```diff
@@ -0,0 +1,284 @@
+#include "meta.hpp"
+#include <cstdint>
+#include <string>
+#include <string_view>
+#include "mgp.hpp"
```

Comment:
newline break after line 4

Reply:
the structure should be 1st the header file of the file followed by std libraries and others, followed by local imports like `mgp.hpp`